core/src/main/scala/HelmOp.scala

```diff
@@ -21,6 +21,14 @@ object ConsulOp {
 
   final case class HealthCheck(service: String) extends ConsulOp[String]
 
+  final case class AgentRegisterService(
+    service: String,
+    id:      Option[String],
+    tags:    Option[NonEmptyList[String]],
+    address: Option[String],
+    port:    Option[Int]
```

Comment:
I wonder if we should support [all the parameters](https://www.consul.io/api/agent/service.html#parameters) supported by `AgentRegisterService` out of the gate to avoid a breaking change later when someone wants to specify one.